source/addonHandler/__init__.py

```diff
@@ -97,7 +97,7 @@ def getIncompatibleAddons(
 
 def completePendingAddonRemoves():
 	"""Removes any add-ons that could not be removed on the last run of NVDA"""
-	user_addons = os.path.abspath(os.path.join(globalVars.appArgs.configPath, "addons"))
+	user_addons = os.path.join(globalVars.appArgs.configPath, "addons")

```

Comment:
It is a little tricky to make this global as addonHandler is imported sometimes earlier than globalVars.appArgs.configPath being set (if it was not specified as a commandline argument). Also my try at changing this was breaking system tests. I'd like to leave it as is in this pr at least as the change would have no functional benefit. But I do think that in the long-term it would be nice to perhaps move away from globalVars and use more module or class-specific variables like this.